### PR TITLE
fallback to 2 evm versions

### PR DESCRIPTION
**Pull request by ayrat555**
fixes https://github.com/poanetwork/blockscout/issues/1653#issuecomment-487034865

### CODE REVIEW

File: apps/explorer/lib/explorer/smart_contract/verifier.ex [hunk 1]
```diff
@@ -17,13 +17,28 @@ defmodule Explorer.SmartContract.Verifier do
     do: {:error, :contract_source_code}
 
   def evaluate_authenticity(address_hash, params) do
+    evm_version = Map.get(params, "evm_version", "petersburg")
```
Comment (vbaranov) [inline apps/explorer/lib/explorer/smart_contract/verifier.ex]: @ayrat555 could we change hardcoded "petersburg" here to the last element from `CodeCompiler.allowed_evm_versions()` ?

Comment (vbaranov) [reply apps/explorer/lib/explorer/smart_contract/verifier.ex]: I think in the future we could move the list of allowed EVM versions to ENV var https://github.com/poanetwork/blockscout/issues/1918

**Comment by vbaranov** [comment]
@ayrat555 btw, would you mind to update CHANGELOG with this the info from this PR?

File: apps/explorer/lib/explorer/smart_contract/verifier.ex (hunk 1 above)
Comment (ayrat555) [reply apps/explorer/lib/explorer/smart_contract/verifier.ex]: done

**Pull request merged**